### DISCUSSION TITLE
Suggestion - FAQ should have link to info about DocC documentation

### DISCUSSION THREAD

**Discussion by ConfusedVorlon** [SwiftPackageIndex/SwiftPackageIndex-Server#2362] [Ideas]
something along the lines of:

Question: How can I get SPI to provide documentation for my package

Answer
- Introductory post here:
https://blog.swiftpackageindex.com/posts/auto-generating-auto-hosting-and-auto-updating-docc-documentation/

- formal documentation here:
https://swiftpackageindex.com/SwiftPackageIndex/SPIManifest/<Latest Version>/documentation/spimanifest/commonusecases

**Comment by daveverwer** [comment]
Thanks @ConfusedVorlon, this is a good idea.

I want to remove that FAQ (even the term is a bit late 90s 😂) and replace it with a Documentation link that links to some DocC documentation that covers the main bits of that which are still useful.

In the meantime, we could add that link as you suggest. I'll promote this to an issue.
